python/lsst/meas/algorithms/defects.py

```diff
@@ -151,6 +151,33 @@ def __eq__(self, other):
     def __str__(self):
         return "Defects(" + ",".join(str(d.getBBox()) for d in self) + ")"
 
+    def normalize(self):
+        """Recalculate defects bounding boxes to use minimal set.
+        A normalized copy is returned.
+
+        Returns
+        -------
+        normalizedDefects : `lsst.meas.algorithms.Defect`
```

Comment:
`Defects` not `Defect` (and I don't think you need to qualify it with `lsst.meas.algorithms`